source/_docs/configuration.markdown

```diff
@@ -40,6 +40,7 @@ If you can't see your integration listed there, you will need to restart Home As
 
 ## Migrating to a new system
 
-The preferred way of migrating to a new system is by {% my supervisor_backups title="making a backup" %}.
+The preferred way of migrating to a new system is by backup using {% my supervisor_backups title="supervisor backup" %}. Then install new system (HAOS or Supervisor), and restore backup at first logon.
```

Comment:
I don't think changing the link to "supervisor backup" is correct, it might be confused with taking a backup of just the supervisor (which is of course not the case).

```suggestion
The preferred way of migrating to a new system is by backup using {% my supervisor_backups title="backup" %}. Then install the new system (HAOS or Supervisor), and restore the backup during the onboarding steps.
```